homeassistant/components/evohome/__init__.py

```diff
@@ -431,11 +432,11 @@ async def call_client_api(self, api_function, refresh=True) -> Any:
                 return
 
         if refresh:
-            self.hass.helpers.event.async_call_later(1, self.async_update())
+            asyncio.create_task(self.async_update(delay=1))
```

Comment:
I'm not sure all of this needs to change?  How about 

```suggestion
            self.hass.helpers.event.async_call_later(1, self.async_update)
```

instead?

Reply:
I noticed another issue where a web API was being called unnecessarily This stops that - see the comment below.